### PR TITLE
Seo rodeo ii Website updates

### PR DESCRIPTION
**Pull request by wtfsayo**

### CODE REVIEW

File: assets/illustrations/fallBackBanner.png
Comment (scottrepreneur) [inline assets/illustrations/fallBackBanner.png:1]: this banner is 3 mb 👀

File: components/landing/2-Manifesto.tsx [hunk 1]
```diff
@@ -1,22 +1,29 @@
...
+import tokens from '../../utils/extendedTokens';
 import Link from '../atoms/ChakraNextLink';
 
 import raidFantasy from '../../assets/illustrations/raid__fantasy.webp';
 
 const SectionTwo = () => {
   return (
     <SimpleGrid
-      layerStyle='redToPurpleHorizontalGradient'
+      bg={tokens.background}
```
Comment (scottrepreneur) [inline components/landing/2-Manifesto.tsx]: don't access tokens directly. use from theme

File: components/landing/3-Services.tsx [hunk 2]
```diff
@@ -1,55 +1,73 @@
...
-    layerStyle='purpleToRedDiagonalGradient'
-    minHeight='95vh'
-    direction={['column', null, null, 'row']}
-    px={{ base: '2rem', lg: '8rem' }}
-    py='2rem'>
-    <VStack spacing={5} justifyContent='center' my='5rem' maxW='500px'>
-      <Heading fontSize={{ base: '1.5rem', lg: '36px' }}>Our Services</Heading>
-      <Text fontSize={{ base: '1rem', lg: '18px' }}>
+  <SimpleGrid
+    bg={tokens.purpleToRedGradient}
```
Comment (scottrepreneur) [inline components/landing/3-Services.tsx]: use layer styles

File: components/landing/6-Supporters.tsx [hunk 3]
```diff
@@ -1,37 +1,83 @@
-import { Flex, Box, Heading } from '@raidguild/design-system';
+import { Box, Center, Link, SimpleGrid, Text, Tooltip } from '@raidguild/design-system';
 import Image from 'next/image';
 
-import metacartel from '../../assets/logos/metacartel.webp';
+import lexdao from '../../assets/logos/LexDAO-Logo.png';
```
Comment (scottrepreneur) [inline components/landing/6-Supporters.tsx]: prefer `public` for resources that don't need to be wrapped in a component (most images)

File: components/page-components/Nav.tsx [hunk 4]
```diff
@@ -1,132 +1,283 @@
...
+                            alignItems='center'
+                            justifyItems='center'
+                            gap={8}
+                            _hover={{
+                              bgColor: `${defaultTheme.colors.primary[500]}40`,
+                              textColor: defaultTheme.colors.primary[500],
+                            }}
+                            w='full'
+                            p={2.5}>
+                            {menuItem.category === 'Development' && <Castle fontSize={28} />}
```
Comment (scottrepreneur) [inline components/page-components/Nav.tsx]: include icon in const array

File: components/atoms/AnimatedButton.tsx [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+    case 'bottom':
+      return bottomVariant;
+    case 'left':
+      return leftVariant;
+    default:
+      return leftVariant;
+  }
+}
+
+const AnimatedButton = ({ children, start = 'left', ...props }: AnimatedButtonProps) => {
```
Comment (scottrepreneur) [inline components/atoms/AnimatedButton.tsx:82]: ds

File: components/atoms/Markdown.tsx [hunk 6]
```diff
@@ -1,15 +1,54 @@
...
+      </Text>
+    );
+  },
+};
 
 type MarkdownProps = {
   children: string;
 };
 
 const Markdown = ({ children }: MarkdownProps) => (
```
Comment (scottrepreneur) [inline components/atoms/Markdown.tsx:50]: ds component

File: components/page-components/Nav.tsx [hunk 7]
```diff
@@ -1,132 +1,283 @@
...
+          ),
+        )}
+      </HStack>
+
+      <ConnectWallet />
+    </HStack>
+  );
+};
+
+const MobileNav = ({ isOpen, setIsOpen }: any) => {
```
Comment (scottrepreneur) [inline components/page-components/Nav.tsx:189]: Should use a `<Drawer />` Component here, no?

File: components/page-components/ProjectCard.tsx [hunk 8]
```diff
@@ -1,41 +1,59 @@
...
-      <Box minHeight='2rem' />
-      <Link href={_.get(project, 'websiteUrl')} isExternal>
-        <Button>Visit Website</Button>
-      </Link>
-    </VStack>
-  </Card>
-);
+const ProjectCard = ({ name, logo, website, roles }: ProjectCardProps) => {
+  return (
+    <Stack maxW='600px' border={`1px solid ${defaultTheme.colors.red[500]}`} align='center' spacing={4} pb={6}>
```
Comment (scottrepreneur) [inline components/page-components/ProjectCard.tsx]: why not use an actual `<Card />` component

File: package.json [hunk 9]
```diff
@@ -8,29 +8,35 @@
...
-    "@rainbow-me/rainbowkit": "^1.0.8",
-    "@rainbow-me/rainbowkit-siwe-next-auth": "^0.3.0",
+    "@raidguild/design-system": "^0.5.7",
+    "@rainbow-me/rainbowkit": "^1.3.0",
+    "@rainbow-me/rainbowkit-siwe-next-auth": "^0.3.2",
     "@tanstack/react-query": "^4.19.1",
     "@tanstack/react-query-devtools": "^4.19.1",
+    "@types/lodash": "^4.14.202",
     "@vercel/analytics": "^1.0.1",
+    "axios": "^1.6.3",
```
Comment (scottrepreneur) [inline package.json:26]: what is axios being used for?

File: pages/_app.tsx [hunk 10]
```diff
@@ -8,18 +8,18 @@ import {
...
 import '@rainbow-me/rainbowkit/styles.css';
 import { QueryCache, QueryClient, QueryClientProvider } from '@tanstack/react-query';
 import { ReactQueryDevtools } from '@tanstack/react-query-devtools';
 import { Analytics } from '@vercel/analytics/react';
+import * as Fathom from 'fathom-client';
 import { SessionProvider } from 'next-auth/react';
 import { DefaultSeo } from 'next-seo';
 import type { AppProps } from 'next/app';
+import { useRouter } from 'next/router';
+import { useEffect } from 'react';
```
Comment (scottrepreneur) [inline pages/_app.tsx:22]: used?

File: pages/services/[service]/index.tsx [hunk 11]
```diff
@@ -0,0 +1,73 @@
...
+
+const Service = ({ title, description, roleImage, salesContent, data, meta }: Props) => {
+  return (
+    <ServicePageTemplate
+      pageTitle={title}
+      pageDescription={description}
+      roleImage={roleImage}
+      salesContent={salesContent}>
+      <Head>
+        <title>{meta?.title}</title>
```
Comment (scottrepreneur) [inline pages/services/[service]/index.tsx]: `next-seo`?

File: utils/constants.ts [hunk 12]
```diff
@@ -34,7 +34,7 @@ export const culture = [
   },
   {
     name: 'Tip of the Spear',
-    img: 'https://res.cloudinary.com/saimano/image/upload/v1622037333/RaidGuild/icons/purple/tipofthespear_k2lkvk.png',
+    img: 'https://cdn.discordapp.com/attachments/1168646704669663323/1171320352694079540/tipofthespear.png',
```
Comment (scottrepreneur) [inline utils/constants.ts]: definitely shouldn't be relying discord as an image provider

File: utils/constants.ts [hunk 13]
```diff
@@ -163,3 +163,67 @@ export const hireUsServices = [
 
 export const messageToSignJoin =
   'I hereby with the information provided would like to submit my cohort application to RaidGuild.';
+
+export const swordsImage = 'https://cdn.discordapp.com/attachments/1168646704669663323/1171323729968627742/swords.png';
```
Comment (scottrepreneur) [inline utils/constants.ts]: discord is not an image host

File: utils/extendedTokens.ts [hunk 14]
```diff
@@ -0,0 +1,13 @@
+const tokens = {
```
Comment (scottrepreneur) [inline utils/extendedTokens.ts]: layer styles

File: package.json (hunk 9 above)
Comment (scottrepreneur) [reply package.json:26]: pinata

File: pages/_app.tsx (hunk 10 above)
Comment (wtfsayo) [reply pages/_app.tsx:22]: umm, yes for fathom analytics

**Pull request merged**